tests/Node/TableNodeTest.php

```diff
@@ -333,9 +348,10 @@ public function testGetTableFromListWithMultidimensionalArrayArgument(): void
             new NodeException('List is not a one-dimensional array.')
         );
 
+        // @phpstan-ignore argument.type (we are explicitly testing an invalid instantiation)
         TableNode::fromList([
-            [1, 2, 3],
-            [4, 5, 6],
+            ['1', '2', '3'],
+            ['4', '5', '6'],
```

Comment:
using string values here inside the array corresponds to using `fromList` with the type expected by the constructor, which I find clearer (any nested array is rejected anyway).